SphereEversion/Loops/Reparametrization.lean

```diff
@@ -1,5 +1,6 @@
 import Mathlib.Analysis.Calculus.BumpFunction.Convolution
 import Mathlib.Analysis.Calculus.BumpFunction.InnerProduct
+import Mathlib.MeasureTheory.Integral.IntegrationByParts
```

Comment:
Could you explain this new import?

Reply:
intervalIntegral.integral_comp_smul_deriv moved there in https://github.com/leanprover-community/mathlib4/pull/22788

Reply:
Ok, the new file name is rather confusing since that lemma has not much to do with integration by parts.